app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayFragment.kt

```diff
@@ -60,12 +60,15 @@ import org.mozilla.fenix.tabstray.ext.collectionMessage
 import org.mozilla.fenix.tabstray.ext.make
 import org.mozilla.fenix.tabstray.ext.orDefault
 import org.mozilla.fenix.tabstray.ext.showWithTheme
+import org.mozilla.fenix.tabstray.syncedtabs.SyncedTabsIntegration
 import org.mozilla.fenix.theme.ThemeManager
 import org.mozilla.fenix.utils.allowUndo
 import kotlin.math.max
 
 @Suppress("TooManyFunctions", "LargeClass")
-class TabsTrayFragment : AppCompatDialogFragment() {
+class TabsTrayFragment :
+    AppCompatDialogFragment() {
```

Comment:
nit: This change seems unnecessary and not something we see a lot of in the codebase.

Reply:
I think this was leftover when TabsTrayFragment had a bunch of other interfaces. I'll clean this up